cmd/backup-manager/app/import/manager.go

```diff
@@ -166,6 +167,8 @@ func (rm *RestoreManager) performRestore(restore *v1alpha1.Restore) error {
 		return errorutils.NewAggregate(errs)
 	}
 	klog.Infof("unarchive cluster %s backup %s data success", rm, restoreDataPath)
+	// unarchive succeed, origin archive can be deleted now
```

Comment:
What about keeping the tar, it does not make much difference since we keep the unarchieved files.

Reply:
addressed in https://github.com/pingcap/tidb-operator/pull/2897/commits/df73f02c027c04d97c78084e29bb5bb59b1bb7f3